crates/runtime/src/lib.rs

```diff
@@ -38,13 +38,13 @@ pub trait SyscallPtrAccess {
 }
 
 // TODO this is only temporary, after we migrate everything to extension it will be auto-derived
-impl<'a> SyscallPtrAccess for ContractExecutionSyscallHandler<'a> {
+impl<'a> SyscallPtrAccess for CheatableSyscallHandler<'a> {
```

Comment:
This can also be done with a macro
I don't think you have to migrate everything to do it

Reply:
I need only the last one that is not migrated yet to have this implemented. After I migrate it fully it will be auto derived